prow/test/integration/test/sinker_test.go

```diff
@@ -142,18 +183,26 @@ func TestDeletePod(t *testing.T) {
 			}); err != nil {
 				t.Fatalf("Pod was not created successfully: %v", err)
 			}
-			t.Logf("Pod is running: %s", tt.name)
-
-			// Delete prowjob if CR isn't supposed to have existed
-			if !tt.hasCR {
-				if err := kubeClient.Delete(ctx, &prowjob); err != nil {
-					t.Fatalf("Failed deleting prowjob: %v", err)
+			t.Logf("Pod is running: %s", resourceName)
+
+			if !tt.notCreatedByProw {
```

Comment:
Why do we update the prowjob later rather than creating it directly with the right settings?

Reply:
This was a side effect of get after create, removed